### DISCUSSION TITLE
FitBit Versa 3 is crap

### DISCUSSION THREAD

**Discussion by smmdolfin424** [community/community#69679] [Other Feature Feedback, Questions, & Ideas]
Any codes or any suggestions on how to get this versa 3 to actually track my sleep and activities ACCURATELY?

**Comment by queenofcorgis** [comment]
Hi @smmdolfin424 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on FitBit like their [community](https://community.fitbit.com/t5/Community/ct-p/EN). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
